plugins/catalog-backend/src/ingestion/processors/github/github.ts

```diff
@@ -241,6 +258,29 @@ export async function getOrganizationTeams(
     { org },
   );
 
+  const organizationEntity: GroupEntity = {
+    apiVersion: 'backstage.io/v1alpha1',
+    kind: 'Group',
+    metadata: {
+      name: orgSlug,
+      namespace: 'default',
```

Comment:
I do think having this in the default namespace is gonna end up being quite awkward. Both when it comes to access control-type things, but also referencing the org and also avoiding conflicts in the default namespace. Perhaps we can figure out same name here that isn't a valid GitHub team name, so that it's safe to place it within the same namespace?